test/test_crc32.cpp

```diff
@@ -46,24 +45,24 @@ TORRENT_TEST(crc32)
 	std::uint32_t in1 = 0x5aa5feef;
 	out = aux::crc32c_32(in1);
 
-	TEST_EQUAL(out, aux::host_to_network(0xd5b9e35eU));
+	TEST_EQUAL(out, 0x5ee3b9d5);
```

Comment:
@arvidn, I hope you don't mind, can you elaborate a little bit here the reason? (it's just me learning)

Reply:
yeah, I'm pretty sure this was a mistake, to expect the result from the CRC32 function to be different depending on endianness. In the two places where this is used, it's used as an integer, not a sequence of bytes. (it's used in the DHT node ID generation and validation and the peer connection affinity).

As such, the result must be the same *integer*, the native endian of integers is not relevant.

Reply:
I see, thanks